### PR TITLE
Add iPad support for screen size

### PR DESCRIPTION
**Pull request by gotvitch**
#15

### CODE REVIEW

File: Example/ViewController.swift [hunk 1]
```diff
@@ -58,11 +58,14 @@ class ViewController: UIViewController {
         
         /*** Display the device screen size ***/
         switch Device.size() {
-            case .Screen3_5Inch: print("It's a 3.5 inch screen")
-            case .Screen4Inch:   print("It's a 4 inch screen")
-            case .Screen4_7Inch: print("It's a 4.7 inch screen")
-            case .Screen5_5Inch: print("It's a 5.5 inch screen")
-            default:             print("Unknown size")
+            case .Screen3_5Inch:  print("It's a 3.5 inch screen")
+            case .Screen4Inch:    print("It's a 4 inch screen")
```
Comment (joeblau) [inline Example/ViewController.swift]: Should this be 4_0 for consistency?

File: README.md [hunk 2]
```diff
@@ -89,11 +89,14 @@ func myFunc() {
 func myFunc() {
         /*** Display the device screen size ***/
         switch Device.size() {
-            case .Screen3_5Inch: print("It's a 3.5 inch screen")
-            case .Screen4Inch:   print("It's a 4 inch screen")
-            case .Screen4_7Inch: print("It's a 4.7 inch screen")
-            case .Screen5_5Inch: print("It's a 5.5 inch screen")
-            default:             print("Unknown size")
+            case .Screen3_5Inch:  print("It's a 3.5 inch screen")
+            case .Screen4Inch:    print("It's a 4 inch screen")
```
Comment (joeblau) [inline README.md]: Here as well although this might break a bunch of peoples code now that I Think about it :)

**Comment by joeblau** [comment]
:+1: Looks great. now we just need @Ekhoo to push the button.

**Comment by Ekhoo** [comment]
@gotvitch Thank you for this PR, I will push a new release as soon as possible with your PR. :+1:

File: Device.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -436,6 +436,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				TARGETED_DEVICE_FAMILY = "1,2";
```
Comment (Ekhoo) [inline Device.xcodeproj/project.pbxproj]: Why ?

Comment (joeblau) [inline Device.xcodeproj/project.pbxproj]: In order for this project to run on the iPad, you need to allow the iPad device target.

| Description: | Comma-separated list of numeric identifiers. Specifies the device families on which the product must be capable of running. |
|:---|:---|
| Identifiers: | 1: iPhone/iPod touch.  2: iPad. |
|Default value: | 1 |
| Example value: | 1,2 |

Source: https://developer.apple.com/library/mac/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW165

Comment (Ekhoo) [inline Device.xcodeproj/project.pbxproj]: :+1:  Thank you :)

**Comment by Ekhoo** [comment]
Device `v1.0.1` is available with your PR. Thank you again :)

**Comment by gotvitch** [comment]
Cool thank you too

**Pull request merged**